code/modules/reagents/newchem/drugs.dm

```diff
@@ -551,7 +551,7 @@
 /datum/reagent/lube/ultra/addiction_act_stage4(var/mob/living/carbon/human/M as mob)
 	M.Jitter(20)
 	M.Dizzy(20)
-	M.adjustToxLoss(5)
+	M.adjustBrainLoss(2)
```

Comment:
Yes they can, Posibrains function as a brain organ for them. Brain damage can be repaired via surgery or via the administering of Liquid Solder.

IPC brain damage was actually quite scary before the IPC update, since it wasn't possible to revive an IPC who died of brain damage. Mark has assured me that it is fixable via surgery though, so that shouldn't be a concern anymore.